src/helpers/navigationRenderer.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import BottomNavigationAction from '@mui/material/BottomNavigationAction'
+import RenderIcon from './renderIcon'
+
+interface NavigationItem {
+  value: string
+  iconProps: any // Adjust the type according to your needs
```

Comment:
type iconProps

Reply:
In navigationRenderer.tsx changed iconProps: any=> {imagePath: string, alt: string, size: {width: number; height: number}